packages/api/models/granules.js

```diff
@@ -471,6 +478,111 @@ class Granule extends Manager {
     return Promise.all(granules.Items.map((granule) =>
       super.delete({ granuleId: granule.granuleId })));
   }
+
+  /**
+   * Get the set of fields which are mutable based on the granule status.
+   *
+   * @param {Object} record - A granule record
+   * @returns {Array} - The array of mutable field names
+   */
+  _getMutableFieldNames(record) {
```

Comment:
This is another candidate for a class method that could really just be a stand-alone function.

Reply:
This felt like a class method to me because it is specific to behavior of storing granule records, which seems like a reasonable if loose definition for this model